cmd/apps/argo_app.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) arkade author(s) 2020. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/k8s"
+
+	"github.com/alexellis/arkade/pkg"
+
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallArgo() *cobra.Command {
+	var command = &cobra.Command{
+		Use:          "argo",
+		Short:        "Install argo",
+		Long:         `Install argo`,
+		Example:      `  arkade install argo`,
+		SilenceUsage: true,
+	}
+
+	command.RunE = func(command *cobra.Command, args []string) error {
+		kubeConfigPath, _ := command.Flags().GetString("kubeconfig")
+		if err := config.SetKubeconfig(kubeConfigPath); err != nil {
+			return err
+		}
+
+		arch := k8s.GetNodeArchitecture()
+		fmt.Printf("Node architecture: %q\n", arch)
+
+		if arch != IntelArch {
+			return fmt.Errorf(OnlyIntelArch)
+		}
+
+		_, err := k8s.KubectlTask("create", "ns",
+			"argo")
+		if err != nil {
+			if !strings.Contains(err.Error(), "exists") {
+				return err
+			}
+		}
+
+		_, err = k8s.KubectlTask("apply", "-f",
+			"https://raw.githubusercontent.com/argoproj/argo-workflows/v2.11.8/manifests/install.yaml", "-n", "argo")
```

Comment:
It would be great to provide a `--version` flag to template in the `v2.11.8` (which could be set as the default, or the latest stable version could be used, and you can set --version for this specific one as indicted in the pr body)

Reply:
Agreed. I see `stable` as a good default if you can set the version.